scalafix-tests/output/src/main/scala-2.11/test/explicitResultTypes/ExplicitResultTypesInfix.scala

```diff
@@ -0,0 +1,11 @@
+package test.explicitResultTypes
```

Comment:
so this one doesn't work yet on scala 2.13?

Reply:
So this one doesn't find the same types in 2.11 (compared to 2.12 and 2.13).
On 2.12 and 2.13, there is still an output test that looks like that
```
object ExplicitResultTypesInfix {
  val ab: Int \/ String = null.asInstanceOf[Int \/ String]
  def foo: Int \/ String = identity(ab)

  case class :+:[T1, T2]()
  def bar: Double :+: Long = :+:[Double, Long]()
}
```

Reply:
makes sense, sorry I ad missed the 3 different outputs!